examples/resources/genesyscloud_tf_export/resource.tf

```diff
@@ -1,9 +1,79 @@
-resource "genesyscloud_tf_export" "export" {
-  directory = "./terraform"
-  // leaving resource_types empty will cause all exportable resources to be exported
-  // export all resources of a single type by providing the resource type
-  // resources can be exported by name with the syntax `resource_type::regular expression`
-  include_filter_resources = ["genesyscloud_user", "genesyscloud_routing_queue::-(dev|test)$"]
+# resource "genesyscloud_tf_export" "export" {
+#   directory = "./terraform"
+#   // leaving resource_types empty will cause all exportable resources to be exported
+#   // export all resources of a single type by providing the resource type
+#   // resources can be exported by name with the syntax `resource_type::regular expression`
+#   include_filter_resources = ["genesyscloud_user", "genesyscloud_routing_queue::-(dev|test)$"]
+#   include_state_file       = true
+#   exclude_attributes       = ["genesyscloud_user.skills"]
+# }
+
+
+terraform {
+  required_providers {
+    genesyscloud = {
+      source  = "mypurecloud/genesyscloud"
+      version = "1.32.0"
+    }
+  }
+}
+
+
+# This will only generate group resources with the string ending with "dev" or "test"
+resource "genesyscloud_tf_export" "include-filter" {
+  directory                = "./genesyscloud/include-filter"
+  export_as_hcl            = true
+  log_permission_errors    = true
   include_state_file       = true
-  exclude_attributes       = ["genesyscloud_user.skills"]
+  include_filter_resources = ["genesyscloud_group::-(agent|jen)$"]
+  # include_filter_resources = ["genesyscloud_routing_queue::Barrera Test Queue"]
+  # include_filter_resources = ["genesyscloud_group"]
+
+}
+
+# This will generate ALL resources in the org except the ones listed
+resource "genesyscloud_tf_export" "exclude-filter" {
+  directory             = "./genesyscloud/exclude-filter"
+  export_as_hcl         = true
+  log_permission_errors = true
+  include_state_file    = true
+  exclude_filter_resources = [
```

Comment:
I don't think this list needs to be so long, people will understand the purpose of the attribute with a few items